app/dashboard/templates/labs.html

```diff
@@ -115,6 +116,12 @@ <h3 class="sub-title dark-text">Austin Griffith</h3>
                 <i class="fab fa-medium fa-lg"></i>
               </a>
             </div>
+
+            <div class="mx-lg-auto col-lg-6 col-xl-3 mt-3 text-center pl-xl-0 ml-xl-0">
+              <h1 class="dark-text">You?</h1>
+              <p class="description">Want to join the Labs team? Start contributing on <a href="https://github.com/gitcoinco/" class="font-weight-bold magenta-link">GitHub</a>. 
```

Comment:
All new copy needs wrapped in `{% trans "" %}`

Reply:
Sure